### DISCUSSION TITLE
Is there an option to mark readers' comments as private as an admin?

### DISCUSSION THREAD

**Discussion by bravking** [Tetrakern/fictioneer#43] [Q&A] [answered]
I know that we can private our own comments but is there an option to mark readers' comments as private?

**Comment by Tetrakern** [comment, marked as answer]
No. I'm also concerned it would be taken wrongly if a comment suddenly becomes privated without the users consent. Obviously, you could just change the comment type to "private", either in the database or via a plugin.
